bom/pom.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.cxf</groupId>
+        <artifactId>quarkus-cxf-parent</artifactId>
+        <version>1.3.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
```

Comment:
BOMs usually don't declare a parent, but I can see why this is needed

Reply:
Yeah, this is related to that inheritance pitfall I mentioned further up.

Without it distributionManagement etc. is missing.

Reply:
Maybe it would be interesting to consider using https://www.mojohaus.org/flatten-maven-plugin/ when releasing